### DISCUSSION TITLE
Error while learning the tutorial

### DISCUSSION THREAD

**Discussion by KGmax** [Open-Systems-Pharmacology/Forum#1534] [Questions & Problems] [answered]
When I was studying the tutorial , an error occurred when generating the simulation. As shown in the picture
![提示错误](https://github.com/Open-Systems-Pharmacology/Forum/assets/107936033/db40432c-cc96-4082-b543-45635f00ebeb)
Negative values appear when the simulation time is set to 8 days. The same error occurs with files downloaded from the website.I am a new user and I don't know how to solve this problem. 
I would appreciate your help, thanks.

**Comment by msevestre** [comment]
@KuoGeng What tutorial are you talking about?

**Reply by msevestre** [reply]
Ok can you save the project, zip it, and upload it here so that someone can potentially take a look?

**Comment by StephanSchaller** [comment, marked as answer]
@KuoGeng, the solution should also be provided in the tutorial. Just continue the tutorial.

**Reply by StephanSchaller** [reply]
@KuoGeng, can you mark this as answered?, Thanks

**Reply by KGmax** [reply]
Thank you very much.
